packages/entity-renderer/env.js

```diff
@@ -1,4 +1,3 @@
-import { create } from "@zazuko/env";
-import DatasetFactory from "rdf-ext/DatasetFactory.js";
+import { create } from '@zazuko/env'
 
-export default create(DatasetFactory);
+export default create()
```

Comment:
If it works (famous last words), this module should be unnecessary and you'd simply use `@zazuko/env` directly.

The bundled dataset now comes with most methods of rdf-ext's dataset implementation.

Reply:
It was working, so I will try by using `@zazuko/env` directly.
I will update this PR with those changes after I do some tests.

Reply:
It's working that way, so we can remove the `env.js` file as you suggested :)